### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by thomasyxy**
update README

### CODE REVIEW

**Comment by hacdias** [comment]
Why did you change archiver's user to your own fork? I thank the changes to the README, but I won't merge it until it is mholt's archiver back, please.

File: download.go
Comment (hacdias) [inline download.go]: This change is making the build to fail.

**Review by hacdias — CHANGES_REQUESTED** [review]
Change github.com/thomasyxy/archiver to github.com/mholt/archiver

**Comment by thomasyxy** [comment]
I'm so sorry, I don't know commit after create a pull requests will take effect.

Sorry about the inconvenience.

**Comment by hacdias** [comment]
No problem :) your commits won't affect only after merging

**Comment by hacdias** [comment]
Thanks 😄

**Pull request merged**
